translations/data/navigation.json

```diff
@@ -1,20 +1,23 @@
 {
    "home": {
         "en-us": "Home",
+	"de-de": "Hauptseite",

```

Comment:
Okay, "Home" is hardly to be translated to "Heimatseite", LOL. But also "Eingangsseite" could work.

Reply:
I like the term "Startseite".